client/lib/abtest/index.js

```diff
@@ -100,17 +134,34 @@ ABTest.prototype.getVariation = function() {
 };
 
 ABTest.prototype.isEligibleForAbTest = function() {
-	var selectedSite = sites.getSelectedSite();
+	const selectedSite = sites.getSelectedSite();
+	const client = ( typeof navigator !== 'undefined' ) ? navigator : {};
```

Comment:
Unfortunately our linting rules don't like that. I get the error `'navigator' was used before it was defined`, when I try that, which I do not get when I leave it as-is in the PR.

![lint error](https://cloudup.com/cuSgo_jRMqg+)